### PR TITLE
update template

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "ecmarkup": "^8.1.0"
+    "@tc39/ecma262-biblio": "latest",
```
Comment (ljharb) [inline package.json]: this should not be using `latest`, it should be using a semver range. v2 of this package has a breaking change compared to v1, for example.

File: spec.emu [hunk 2]
```diff
@@ -5,7 +5,7 @@
 <script src="./spec.js"></script>
 <pre class="metadata">
 title: Proposal Title Goes Here
-stage: -1
+stage: 0
```
Comment (ljharb) [inline spec.emu]:
```suggestion
stage: -1
```
it's not stage 0 until it's presented or on an agenda; i'd prefer that to be an explicit action.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Jack-Works) [inline package.json]:
```suggestion
    "@tc39/ecma262-biblio": "^2",
```

**Review by ljharb — APPROVED** [review]
Thanks!

**Pull request merged**